Sorry: TabError: inconsistent use of tabs and spaces in indentation (hw_status_publisher.py, line 18)
